grid2op/tests/test_ma.py

```diff
@@ -170,7 +174,8 @@ def test_build_subgrid_obj(self):
         assert ma_env._subgrids_cls['action']['agent_1'].n_gen == 3
         
         self.check_subgrid_consistency(ma_env, action_domains)
-        
+        assert ma_env.agents == ['agent_0', 'agent_1']
+        assert ma_env.agent_order == ma_env.agents
```

Comment:
missing a test that if you add the agent in the other order in the dictionnary, then it works too

Reply:
done

Reply:
This is not what I meant at all...

I meant: "initialize your action domain with

```
        self.action_domains = {
            'agent_1' : [5,6,7,8,9,10,11,12,13],
            'agent_0' : [0,1,2,3, 4],
        }
        }
```

and redo this test, this tests nothing.

But that's not my problem and I hope you won't run into issues due to that